src/main/java/com/answerdigital/answerking/controller/CategoryController.java

```diff
@@ -50,7 +49,7 @@ public CategoryController(final CategoryService categoryService) {
                 content = { @Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class)) })
     })
     @PostMapping
-    public ResponseEntity<CategoryResponse> addCategory(@Valid @RequestBody final AddCategoryRequest categoryRequest) {
+    public ResponseEntity<CategoryResponse> addCategory(@Valid @RequestBody final RequestModelsCategory categoryRequest) {
```

Comment:
I think in API contract it is called Category.Request which we can name as CategoryRequest class in our codebase.

Reply:
RequestModelsCategory changed to CategoryRequest.